README.md

```diff
@@ -15,7 +15,7 @@ conda activate tabsyn
 Install pytorch
 
 ```
-pip install torch torchvision torchaudio
+conda install pytorch==2.0.1 torchvision==0.15.2 torchaudio==2.0.2 pytorch-cuda=11.7 -c pytorch -c nvidia
```

Comment:
It seems this PR is not relevant anymore. In the main branch, the installation via conda is already provided now.